pkg/ossm/smcp_annotation.go

```diff
@@ -90,17 +90,35 @@ func TestSMCPAnnotations(t *testing.T) {
 		}
 		util.CheckPodRunning("bookinfo", "app=env")
 		time.Sleep(time.Duration(10) * time.Second)
-
-		msg, _ := util.Shell(`kubectl get po -n bookinfo -l app=env -o yaml | grep "annotation-from-smcp"`)
-
-		if !strings.Contains(msg, `test1.annotation-from-smcp: test1`) {
+		podName, _ := util.GetPodName("bookinfo", "app=env")
+		annotations, err := util.GetPodAnnotations("bookinfo", podName, 30)
+		if err != nil {
+			t.Errorf("Failed to get annotations: %v", err)
+			t.Fatal(err)
```

Comment:
Couldn't you replace these 2 lines with just `t.Fatalf("Failed to get annotations: %v", err)`?

Reply:
Sure, I'll delete the second one